step.yml

```diff
@@ -116,3 +116,17 @@ inputs:
       $BITRISE_DEPLOY_DIR/app-debug.apk|mycompany_debug_app.apk
       $BITRISE_DEPLOY_DIR/app-debug-androidTest.apk|mycompany_debug_app_test.apk
       ```
+- api_base_url: "https://api.github.com"
```

Comment:
Please make it required, because the `github.NewEnterpriseClient` requires it.
https://github.com/bitrise-steplib/steps-github-release/blob/781fa9dd1859d76bd81e18fb25ba707fa780253c/main.go#L108

Reply:
Fixed.